sdk/core/azure-core/inc/azure/core/operation_status.hpp

```diff
@@ -24,28 +24,28 @@ namespace Azure { namespace Core {
 
   public:
     /**
-     * @brief Construct a #OperationStatus with \p value.
+     * @brief Construct an #OperationStatus with \p value.
      *
      * @param value A non-absent value to initialize with.
      */
     explicit OperationStatus(const std::string& value) : m_value(value) {}
     /**
-     * @brief Construct a #OperationStatus with \p value.
+     * @brief Construct an #OperationStatus with \p value.
      *
-     * @param value A non-absent value to initialize with.
+     * @param value The value to initialize with.
```

Comment:
Make a similar fix for the two other ctors above:
https://github.com/Azure/azure-sdk-for-cpp/blob/cb32cbeeb942d3dd3f1d3490febf68c76206ff0c/sdk/core/azure-core/inc/azure/core/operation_status.hpp#L29-L41

Reply:
good catch, fixed